app/javascript/dashboard/mixins/specs/agentFixtures.js

```diff
@@ -20,6 +20,36 @@ export default {
       name: 'Samuel Keta',
       role: 'agent',
     },
+    {
+      account_id: 1,
```

Comment:
Change the availability_status default order (online, busy, offline) of this fixture and see specs are running or not.